app/helpers/practical_supports_helper.rb

```diff
@@ -1,2 +1,22 @@
 module PracticalSupportsHelper
+  def practical_support_options(current_value = nil)
+    standard_options = [
+      [ t('patient.helper.practical_support.travel_to_area'), 'Travel to area' ],
+      [ t('patient.helper.practical_support.travel_inside_area'), 'Travel inside area' ],
+      [ t('patient.helper.practical_support.lodging'), 'Lodging' ],
+      [ t('patient.helper.practical_support.companion'), 'Companion' ],
+    ]
+    full_set = [nil] + standard_options + Config.find_or_create_by(config_key: 'practical_support').options
+    if current_value.present? && full_set.map { |opt| opt.is_a?(Array) ? opt[-1] : opt }.exclude?(current_value)
```

Comment:
Do we need this check? It seems to me we could remove it and just lean on the uniq call.

Reply:
Probably true, I'll try just uniqing and see if tests still pass. Good catch.